src/NHibernate.Test/NHSpecificTest/NH1612/NativeSqlCollectionLoaderFixture.cs

```diff
@@ -39,8 +39,8 @@ public void LoadCompositeElementsWithWithSimpleHbmAliasInjection()
 			Country country = LoadCountryWithNativeSQL(CreateCountry(stats), "LoadAreaStatisticsWithSimpleHbmAliasInjection");
 
 			Assert.That(country, Is.Not.Null);
-			Assert.That((ICollection) country.Statistics.Keys, Is.EquivalentTo((ICollection) stats.Keys), "Keys");
-			Assert.That((ICollection) country.Statistics.Values, Is.EquivalentTo((ICollection) stats.Values), "Elements");
+			Assert.That(country.Statistics.Keys, Is.EquivalentTo(stats.Keys), "Keys");
+			Assert.That(country.Statistics.Values, Is.EquivalentTo(stats.Values), "Elements");
```

Comment:
Those casts were likely required for an older NUnit. Since I have not additionally implemented `ICollection` on the map values wrapper, they were failing. We will probably have to document that as a possible breaking change.